.codecov.yml

```diff
@@ -233,11 +237,19 @@ flags:
     paths:
     - nvml/datadog_checks/nvml
     - nvml/tests
+<<<<<<< HEAD
+=======
```

Comment:
```suggestion
```

Reply:
Thank you.  I also fixed the alpha order